demos/superviseCalib/main.yml

```diff
@@ -13,8 +13,8 @@ x-yarp-base: &yarp-base
   volumes:
     - "${HOME}/${YARP_CONF_PATH}:/root/.config/yarp"
     - "${APPSAWAY_CALIB_CONTEXT}:/usr/local/share/iCub/contexts/cameraCalibration/:rw"
-    - "${APPSAWAY_CALIB_CONTEXT}/icubEyes.ini:/usr/local/share/ICUBcontrib/robots/${YARP_ROBOT_NAME}/icubEyes.ini:rw"
-    - "${ICUB_EYES_PATH:-/dev/null}:/usr/local/share/iCub/contexts/cameraCalibration/customEyes.ini:rw"
+    - "${APPSAWAY_CALIB_CONTEXT}/icubEyes.ini:/usr/local/share/ICUBcontrib/robots/${YARP_ROBOT_NAME}/icubEyes.ini"
+    - "${ICUB_EYES_PATH:-/dev/null}:/usr/local/share/iCub/contexts/cameraCalibration/customEyes.ini"
```

Comment:
Aren't these files changed by the containers? Bear in mind that if the file is changed inside the container, its permissions are also changed, and so the host machine will no longer be able to open the file without sudo.

Reply:
These files are changed by the containers, but we verified that if the file is already there the container does not change its permissions.
The file has root permissions only if created by the container (this is why we now only update the permissions of files created by the container).